src/main/java/org/folio/circulation/domain/policy/LoanPolicy.java

```diff
@@ -91,9 +95,20 @@ public static LoanPolicy unknown(String id) {
         errors.add(loanPolicyValidationError("loan is not renewable"));
         return failedValidation(errors);
       }
+      boolean isRenewalWithHoldRequest = false;
+      //Here can be either Hold request or null only
+      if (isHold(firstRequest)) {
+        if (!isHoldRequestRenewable()) {
+          String reason = "Items with this loan policy cannot be renewed when there is an active, pending hold request";
```

Comment:
@Anatolii-Starkov, maybe we can declare this variable as a 'static final' constant?